bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngWig",
-  "version": "3.0.0-rc1",
+  "version": "3.0.1",
```

Comment:
I think it still needs to be RC (release candidate), still a lot needs to be fixed before final release